lib/ansible/modules/database/postgresql/postgresql_publication.py

```diff
@@ -521,9 +522,8 @@ def __pub_set_param(self, param, value, check_mode=False):
         Returns:
             True if successful, False otherwise.
         """
-        query = ("ALTER PUBLICATION %s SET (%s = '%s')" % (pg_quote_identifier(self.name, 'publication'),
-                                                           param, value))
-        return self.__exec_sql(query, check_mode=check_mode)
+        query = ("ALTER PUBLICATION %s SET (%s = " % (pg_quote_identifier(self.name, 'publication'), param))
```

Comment:
To make this really secure, `param` should be quoted as well. Can `pg_quote_identifier` be used here?

Reply:
no, it can't. I'm going to come up with another mechanism to do it later. after the rest